plot-lib/plot/private/no-gui/plot2d.rkt

```diff
@@ -131,6 +133,78 @@
                    #:title title #:x-label x-label #:y-label y-label #:legend-anchor legend-anchor)))
       width height))
 
+(:: legend-pict
+    (->* [(Treeof (U renderer2d nonrenderer))]
+         [#:x-min (U Real #f) #:x-max (U Real #f)
+          #:y-min (U Real #f) #:y-max (U Real #f)]
+         Pict))
+(define (legend-pict renderer-tree
```

Comment:
The name `legend-pict` is a bit confusing (in my opinion anyway).  I think `plot-legend-as-pict` would be a better option, but perhaps there are other possibilities?

I also feel that using the `x-min` ... arguments is a bit counter-intuitive, but I understand why it is necessary.